pkg/dependencies/factory.go

```diff
@@ -827,15 +830,16 @@ func (f *Factory) WithGitOpsFlux(clusterConfig *v1alpha1.Cluster, fluxConfig *v1
 }
 
 func (f *Factory) WithPackageInstaller(spec *cluster.Spec, packagesLocation string) *Factory {
-	f.WithHelmInsecure().WithKubectl()
+	f.WithHelmInsecure().WithKubectl().WithPackageControllerClient(spec).WithPackageClient()
```

Comment:
if it doesn't use helm, should be remove it from here?
```suggestion
	f.WithKubectl().WithPackageControllerClient(spec).WithPackageClient()
```